### ISSUE TITLE
PoseC3D Results are Lower Compared to Paper

### ISSUE THREAD

**Issue by taliegemen** [open-mmlab/mmaction2#1940]
Hello,
I am currently trainining the PoseC3D algorithm. My question is even if i use lr for 1 gpu by linear scaling rule, or use your learning rate I could only achieve top1 result of 0.81. In the paper and model zoo you mention top 1 accuracy of 0.86. How can I achieve this score, am I doing something different?
fyi i am using the config file of `slowonly_r50_u48_240e_ntu120_xsub_keypoint`, with the provided pkl files for train and validation.
I am using it with 24 max epoch, validate each epoch and used learning rates of 0.2 (provided with config file), 0.4 (from mmaction2 log) and 0.05 (by linear scaling rule, assuming you used 8 gpus). I am making trainings in google colab pro with P100 as gpu.
Thanks!

**Comment by taliegemen** [comment]
fyi i am using the config file of `slowonly_r50_u48_240e_ntu120_xsub_keypoint`
I am using it with 24 max epoch, test each epoch and used learning rates of 0.2 (provided with config file), 0.4 (from mmaction2 log) and 0.05 (by linear scaling rule, assuming you used 8 gpus). I am making trainings in google colab pro with P100 as gpu.

**Comment by kennymckormick** [comment]
Here is the checklist: 
1. If I understand correctly, you are sill training for 240 epochs (we use a repeat dataset, and it repeat 10 times, so its 24 x 10 = 240 epochs). If so, go to 2. If not, please train 240 epochs instead. 
2. Perhaps you are using validation accuracy instead testing accuracy (but I think there will be only 1% or 2% Top-1 difference, not that large). 
BTW, I have never tried use linear-scaling to train with 1 GPU, I will have a try to see if your problem can be reproduced.

**Comment by taliegemen** [comment]
Hello again,
With 24 accuracy and validating every epoch i got top1_acc: 0.9094, top5_acc: 0.9906 during training Epoch [24][3840/3940], is that test accuracy? After that when I evaluate i get top1_acc : 0.8104 top5_acc: 0.9710. I am also attaching my training log in the link below. https://pastebin.mozilla.org/SeTH2ki7
Also, what does repeated dataset mean in this example? Should not i get slightly better result with 240 epoch total 10 epoch validation instead of 24 to 1?
Thanks!

**Comment by kennymckormick** [comment]
1. Definitely, "Epoch [24][3840/3940]" is not testing accuracy, it's training accuracy/
2. Based on the log you pasted, you are training for 24 epochs instead of 240, so the performance might be inferior, we officially train 240 epochs.

**Comment by taliegemen** [comment]
Thank you very much!
